app/kafka/manager/KafkaManager.scala

```diff
@@ -232,13 +232,38 @@ class KafkaManager(akkaConfig: Config) {
           Future.successful[ApiError \/ TopicIdentity](-\/(ApiError(s"Topic not found $topic for cluster $clusterName")))
         } { td =>
           tryWithKafkaManagerActor(KMClusterQueryRequest(clusterName, KSGetBrokers)) { brokerList: BrokerList =>
-            TopicIdentity.from(brokerList, td)
+            val metrics = getTopicMetrics(clusterName, topic)
+            TopicIdentity.from(brokerList, td, Some(metrics))
           }
         }
       })
     }
   }
 
+  def getTopicMetrics(clusterName: String, topic: String) : TopicMetrics = {
```

Comment:
We just be able to create a JMX stats collecting actor inside the KafakStateActor by passing in the brokersPathCache.  This way it will be able to get the current list of broker hosts and using the clusterconfig to get JMX port, poll the brokers.